frog/imports/client/AccountModal/Login.js

```diff
@@ -20,17 +22,28 @@ const styles = (theme: Object) => ({
   },
   avatar: {
     margin: theme.spacing(1),
-    backgroundColor: theme.palette.secondary.main
+    backgroundColor: primaryColor
   },
   form: {
     width: '100%', // Fix IE 11 issue.
     marginTop: theme.spacing(1)
   },
   submit: {
-    margin: theme.spacing(2, 0, 2, 0)
+    background: primaryColor,
+    margin: theme.spacing(2, 0, 2, 0),
```

Comment:
NIT: `theme.spacing(2, 0)`

Reply:
Was part of the old code. Somehow missed this change :) Thanks. Resolved.